frontend/packages/knative-plugin/src/utils/dynamic-channel-utils.ts

```diff
@@ -0,0 +1,6 @@
+import { EventingChannelModel, EventingKafkaChannelModel, EventingIMCModel } from '../models';
+
+export const getChannelModelForKind = (kind: string) =>
+  [EventingChannelModel, EventingIMCModel, EventingKafkaChannelModel].find(
```

Comment:
can we have this util to check wrt all all channels loaded dynamically , WDYT?

Reply:
Removed this function and used utils from `dynamic-eventsource-utils`.